addons/website_event_sale/static/src/scss/website_event_sale_templates.scss

```diff
@@ -11,6 +11,6 @@
         text-decoration: underline;
     }
     .dropdown-item {
-        font-size: small;
+        @include font-size(small);
```

Comment:
Same question for such things, is there a mapping from small to px being done by the mixin?